NetKAN/ProceduralFairings.netkan

```diff
@@ -1,16 +1,7 @@
 {
-    "spec_version": 1,
+    "spec_version": "v1.4",
     "identifier": "ProceduralFairings",
-    "$kref": "#/ckan/github/rsparkyc/ProceduralFairings",
-    "$vref": "#/ckan/ksp-avc",
-    "x_netkan_epoch": 1,
-    "name": "Procedural Fairings",
-    "abstract": "Fairings that automatically reshape for any attached payload",
-    "license": "CC-BY",
-    "resources" : {
-        "homepage" : "http://forum.kerbalspaceprogram.com/index.php?/topic/161733-13-1"
-    },
-    "suggests": [
-        { "name" : "PFTextures" }
-    ]
+    "$kref": "#/ckan/netkan/https://raw.githubusercontent.com/KSP-RO/ProceduralFairings/v1.6.1/ProceduralFairings.netkan",
```

Comment:
This URL probably shouldn't have v1.6.1 in it. When you release a new version, either the URL would no longer make sense or you would have to update it with a PR here, somewhat defeating the purpose of a metanetkan.

Reply:
https://raw.githubusercontent.com/KSP-RO/ProceduralFairings/ProceduralFairings.netkan isn't a valid URL. 404.